### ISSUE TITLE
cannot open Preference

### ISSUE THREAD

**Issue by terryching** [spyder-ide/spyder#5800]
## Description

**What steps will reproduce the problem?**

1. 
2. 
3. 

**What is the expected output? What do you see instead?**

**Please provide any additional information below**

  File "C:\Users\Liver\Anaconda2\lib\site-packages\spyder\app\mainwindow.py", line 2609, in edit_preferences
    widget.initialize()
  File "C:\Users\Liver\Anaconda2\lib\site-packages\spyder\plugins\configdialog.py", line 77, in initialize
    self.load_from_conf()
  File "C:\Users\Liver\Anaconda2\lib\site-packages\spyder\plugins\configdialog.py", line 381, in load_from_conf
    color, bold, italic = self.get_option(option, default)
TypeError: 'NoneType' object is not iterable

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 2.7.14
* Qt Versions: 5.6.2, PyQt5 5.6 on Windows

## Dependencies
```
pyflakes >=0.5.0  :  1.6.0 (OK)
pycodestyle >=2.3 :  2.3.1 (OK)
pygments >=2.0    :  2.2.0 (OK)
pandas >=0.13.1   :  0.20.3 (OK)
numpy >=1.7       :  1.13.3 (OK)
sphinx >=0.6.6    :  1.6.3 (OK)
rope >=0.9.4      :  0.10.5 (OK)
jedi >=0.9.0      :  0.10.2 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
sympy >=0.7.3     :  1.1.1 (OK)
cython >=0.21     :  0.26.1 (OK)
qtconsole >=4.2.0 :  4.3.1 (OK)
IPython >=4.0;<6.0:  5.4.1 (OK)
pylint >=0.25     :  1.7.4 (OK)

```

**Comment by CAMGerlach-Baron** [comment]
Looks like it could be a problem loading something from your Spyder settings file. Assuming you've gotten it to work previously, make a backup of your Spyder preferences file (`C:/Users/Liver/.spyder-py2/spyder.ini`) and try resetting your spyder settings with `spyder --reset` from the anaconda prompt. If Spyder then works, the problem was specific to something malformed in your preferences file. If not, then try reinstalling. 

Other than that, more experienced individuals than I have might have more specific help or be able to isolate this as an actual bug; with regard to that, it would be helpful to know what preferences you changed, if any, or if there is anything nonstandard about your environment/usage. Again, make sure you keep a copy of the ini file so if the malformed data occurred on Spyder's side, it can be tracked down. Thanks.

**Comment by ccordoba12** [comment]
This error was already solved in it'll be part of our 3.2.5 version. In the meantime, please run

    spyder --reset

in a terminal (cmd.exe) to solve it.

**Comment by terryching** [comment]
Thanks! Running spyder-reset works.
